src/admin/pupil-collection/views/PupilCollectionsOverview.tsx

```diff
@@ -294,9 +294,13 @@ const PupilCollectionsOverview: FunctionComponent<RouteComponentProps & UserProp
 
 		switch (columnId) {
 			case 'title':
-				// TODO link to correct edit page for pupil collection
 				return (
-					<Link to={buildLink(APP_PATH.ASSIGNMENT_EDIT.route, { id })}>
+					<Link
+						to={buildLink(APP_PATH.ASSIGNMENT_PUPIL_COLLECTION_DETAIL.route, {
+							assignmentId: assignment?.id,
```

Comment:
assignmentId is leeg bij mij, genereert links als:
http://localhost:8080/werkruimte/opdrachten//antwoorden/846bdb27-729b-49bc-b3bb-82707675c4da

Had nochtans proxy gepulled en herstart. Misschien eens dubbelchecken?

Reply:
klopt, de whitelist was overschreven door een andere PR denk ik
Fixed en pushed naar de epic van de proxy